commands_to_load/Mod.py

```diff
@@ -82,5 +83,375 @@ def __init__(self, bot):
+            logger.info('[Mod propagatemute()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+
+        # Get guild
+        channels = ctx.guild.channels
+
+        # Set up the perms overwrite
+        overwrite = discord.PermissionOverwrite()
+        setattr(overwrite, 'send_messages', False)
+        setattr(overwrite, 'manage_messages', False)
+        setattr(overwrite, 'manage_channels', False)
+        setattr(overwrite, 'manage_guild', False)
+        setattr(overwrite, 'manage_nicknames', False)
+        setattr(overwrite, 'manage_roles', False)
+        
+        # Loop through channels and change the perms
+        for channel in channels: 
+            if channel.id not in adminChannels:
+                await channel.set_permissions(MUTED_ROLE, overwrite=overwrite)
+
+        eObj = em(description='Muted permissions spread though all channels like herpies. Enjoy :)', footer='Moderator action')
+        await ctx.send(embed=eObj, delete_after=3.0)
+
+    @commands.command()
+    async def slowmode(self, ctx, time = 10): 
+        logger.info('[Mod slowmode()] slowmode function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod slowmode()] invoking message deleted')
+
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod slowmode()] unathoriezed command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+         
+        await ctx.message.channel.edit(slowmode_delay=time, reason=None)
+        logger.info('[Mod slowmode()] slowmode enable on channel: ' + str(ctx.message.channel) + ', time between messages set to ' + str(time))
+
+    @commands.command()
+    async def makechannel(self, ctx, name, secret = 0):
+        logger.info('[Mod makechannel()] makechannel function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod makechannel()] invoking command deleted')
+
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod makechannel()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        # Verify args
+        if not name: 
+            eObj = em(description='Missing arguments', footer='Error in makechannel command')
+            ctx.send(embed=eObj)
+            return
+
+        # Get the MUTED role
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+
+        overwrite = {
+            ctx.guild.default_role : discord.PermissionOverwrite(mention_everyone=False),
+            MUTED_ROLE : discord.PermissionOverwrite()
+        }

```

Comment:
how come you dont just cover `mention_everyone` below with `setattr`  along with the other functions?